src/asm3/media.py

```diff
@@ -908,3 +925,68 @@ def scale_all_pdf(dbo):
     asm3.al.debug("scaled %d of %d pdfs" % (total, len(mp)), "media.scale_all_pdf", dbo)
 
 
+def watermark_with_transparency(imagedata, animalName):
+    """
+    Watermark the image with animalName and logo. 
+    """
+    try:
+        inputd = asm3.utils.bytesio(imagedata)
+        base_image = Image.open(inputd)
+
+        watermark = Image.open("/var/www/vhosts/asm.maarcadopt.org/watermark.png")
```

Comment:
Ideally, the watermark image should be in the dbfs, so the user can upload watermark.png under Settings->Reports->Extra Images (no changes needed). 

Image data can then be loaded with:
```
asm3.dbfs.get_string_filepath("/reports/watermark.png")
```

Reply:
Your welcome to make that change to the code. However, I did update the code in my 3rd and 4th commits to use sitedef to vs hard coding the files.